index.html

```diff
@@ -123,13 +124,66 @@ <h5><a target="_blank" href="https://github.com/pcengines/apu2-documentation/blo
 
 
         <div class="container">
-            <!-- mainline releases v4.8.0.2 -->
```

Comment:
@mscibisz if each release has this html comment for better visibility, do not delete it for 4.8.0.2 please